stable/insights-chart/values.yaml

```diff
@@ -1,7 +1,7 @@
 # Copyright (c) 2021 Red Hat, Inc.
 # Copyright Contributors to the Open Cluster Management project
 
-org: open-cluster-management
+org: stolostron
```

Comment:
Same as this comment on search-chart.  https://github.com/stolostron/search-chart/pull/124/files#r794021260

Reply:
Updated this